.husky/commit-msg

```diff
@@ -1,4 +1 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
-
-npx commitlint --edit $1
+commitlint --edit $1
```

Comment:
Will it work if `commitlint` is not installed locally?

Reply:
It throws error if it's not installed like this:

```
.husky/commit-msg: line 1: commitlint: command not found
husky - commit-msg script failed (code 127)
husky - command not found in PATH=...
```

But, `commitlint`  and `lint-staged`  are installed because they are direct dependencies. Should I add `npx`  as prefix anyway?